steampy/client.py

```diff
@@ -165,6 +165,9 @@ def get_trade_offer(self, trade_offer_id: str, merge: bool = True) -> dict:
     @login_required
     def accept_trade_offer(self, trade_offer_id: str) -> dict:
         partner = self._fetch_trade_partner_id(trade_offer_id)
+        if partner == False:
```

Comment:
Delete this, because it will raise exception. You can also add to documantation that accept_trade_offer raises exception.